cmd/ppd/terminal.go

```diff
@@ -153,6 +154,60 @@ func terminal(cmd *cobra.Command, args []string) {
 		return event.Activate(amount, fee, gas) == nil
 	}
 
+	updateStake := func(line string, param []string) bool {
+		stakeDelta := int64(1000000000)
+		fee := int64(10000)
+		gas := int64(1000000)
+		incrStake := true
+		if len(param) > 4 {
+			fmt.Println("Expecting at most 4 params. Input amount of stakeDelta, fee amount, gas amount and flag of incrStake(0 for desc, 1 for incr)")
+			return false
+		}
+
+		var err error
+		if len(param) > 0 {
```

Comment:
Is it a good practice to have default value the last three param? It's better to always enforce the 4 params explicitly

Reply:
referred to impl of activate func, i agree it's better to force 4 params here

Reply:
Please change both.

Reply:
done